README.md

```
@@ -1,40 +1,74 @@
+```
+
+In applicable component:
+
+```javascript
+import { showUp } from "@/library/animations";
+
+<Container.Animated {...showUp} />;
 ```
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+## Cloudinary:
```

Markdownlint

Punctuation: ':'
Trailing punctuation in heading

(MD026, no-trailing-punctuation)